atomic-structure/content.md

```diff
@@ -0,0 +1,25 @@
+# Radioactive Decay
+#### Learning objective
+Basic understanding of atomic structure
+
+How atomic structure relates to radioactivity
+___
+#### Overview
+The most straightforward conceptualization of the atom is the shell model, first proposed by Niels Bohr.
+
+The nucleus, containing positively charged protons and neutrons, which carry no charge, is surrounded by shells of negatively charged electrons. Each shell can only hold a fixed number of electrons, and each shell essentially represents a principal energy level. The electrons orbit around the nucleus. 
+
+Radioactive decay therefore is the spontaneous emission of different particles from an unstable (energy state) atom. This can be the emission of an electron, for example. The atom wants to achieve a stable state, but in the process may become charged due to the method of decay. Radioactivity then refers to the energy emitted as a result of decay.
+
+(Quantum physics has shown this is more of an electron cloud, and there is a limit to how preciseeee one can simulataneously know the position or momentum of a particle; aka the
+[Heisenberg Uncertainty Principle](https://goo.gl/images/at7YZA). For now though, we are only concerned with the Bohr shell model.)
+___
```

Comment:
May not need this if you skip a space and reduce the heading hashes for the following heading.